### PR TITLE
{devel}[GCCcore/8.3.0] CMake v3.15.3

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node370.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/6cdcef80b00650b969c13894c24f8313 for a full test report.

File: easybuild/easyconfigs/c/CMake/CMake-3.15.3-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+]
+
+dependencies = [
+    ('ncurses', '6.1'),
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.8'),
+    ('cURL', '7.66.0'),
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.1.1a'),
```
Comment (Micket) [inline easybuild/easyconfigs/c/CMake/CMake-3.15.3-GCCcore-8.3.0.eb]: Should be 1.1.1d

File: easybuild/easyconfigs/c/CMake/CMake-3.15.3-GCCcore-8.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+configopts += '-DZLIB_LIBRARY_RELEASE=$EBROOTZLIB/lib/libz.%s ' % SHLIB_EXT
+configopts += '-DCMAKE_USE_SYSTEM_CURL=1 '
+configopts += '-DCURL_INCLUDE_DIR=$EBROOTCURL/include '
+configopts += '-DCURL_LIBRARY=$EBROOTCURL/lib/libcurl.%s ' % SHLIB_EXT
+
+sanity_check_paths = {
+    'files': ["bin/%s" % x for x in ['ccmake', 'cmake', 'cpack', 'ctest']],
+    'dirs': [],
+}
+
```
Comment (Micket) [inline easybuild/easyconfigs/c/CMake/CMake-3.15.3-GCCcore-8.3.0.eb]: The PR by boegel has sanity_check_commands as well, which seems like a good idea. https://github.com/easybuilders/easybuild-easyconfigs/pull/8962/files

**Comment by smoors** [comment]
duplicate of #8962

**Pull request closed without merging**